sdk/maps/Azure.Maps.Rendering/tests/Samples/RenderClientSamples.cs

```diff
@@ -0,0 +1,211 @@
+            #endregion
+
+            Assert.IsNotNull(image);
+        }
+
+        [Test]
+        public void RenderingStaticImagesWithPinsAndPaths()
+        {
+            TokenCredential credential = new DefaultAzureCredential();
+            string clientId = TestEnvironment.MapAccountClientId;
+            MapsRenderClient client = new MapsRenderClient(credential, clientId);
+            Assert.IsNotNull(client);
+
+            #region Snippet:RenderStaticImagesWithPinsAndPaths
+            // Prepare pushpin styles
+            ImagePushpinStyle pushpinSet1 = new ImagePushpinStyle(
+                new List<PushpinPosition>()
+                {
+                    new PushpinPosition(13.35, 52.577, "Label start"),
+                    new PushpinPosition(13.2988, 52.6, "Label end"),
+                }
+            )
+            {
+                PushpinScaleRatio = 0.9,
+                PushpinColor = Color.Red,
+                LabelColor = Color.Blue,
+                LabelScaleRatio = 18
+            };
+            ImagePushpinStyle pushpinSet2 = new ImagePushpinStyle(
+                new List<PushpinPosition>() {new PushpinPosition(13.495, 52.497, "Label 3")}
+            )
+            {
+                PushpinScaleRatio = 1.2,
+                PushpinColor = Color.Beige,
+                LabelColor = Color.White,
+                LabelScaleRatio = 18
+            };
+
+            // Prepare path styles
+            ImagePathStyle path1 = new ImagePathStyle(
+                new List<GeoPosition>() {
+                    new GeoPosition(13.35, 52.577),
+                    new GeoPosition(13.2988, 52.6)
+                }
+            )
+            {
+                LineColor = Color.Beige,
+                LineWidthInPixels = 5
+            };
+
+            // Prepare static image options
+            GetMapStaticImageOptions staticImageOptions = new GetMapStaticImageOptions(new GeoBoundingBox(13.228, 52.4559, 13.5794, 52.629))
+            {
+                MapImageLayer = MapImageLayer.Basic,
+                MapImageStyle = MapImageStyle.Dark,
+                ZoomLevel = 10,
+                RenderLanguage = "en",
+                ImagePushpinStyles = new List<ImagePushpinStyle>() { pushpinSet1, pushpinSet2 },
+                ImagePathStyles = new List<ImagePathStyle>() { path1 },
+            };
+
+            // Get static image
+            Response<Stream> image = client.GetMapStaticImage(staticImageOptions);
+
+            // Prepare a file stream to save the imagery
+            using (FileStream fileStream = File.Create(".\\BerlinStaticImageWithPinsAndPaths.png"))
+            {
+                image.Value.CopyTo(fileStream);
+            }
+            #endregion
+
+            Assert.IsNotNull(staticImageOptions);
+            Assert.IsNotNull(image);
+        }
+
+        [Test]
+        public void RenderingMapTiles()
+        {
+            TokenCredential credential = new DefaultAzureCredential();
```

Comment:
These all should be `TestEnvironment.Credential` unless contained within an `#if SNIPPET` scope.